pandas-stubs/core/series.pyi

```diff
@@ -208,55 +209,52 @@ class Series(IndexOpsMixin, NDFrame, Generic[S1]):
     _ListLike: TypeAlias = ArrayLike | dict[_str, np.ndarray] | list | tuple | Index
     __hash__: ClassVar[None]
 
+    # TODO: can __new__ be converted to __init__? Pandas implements __init__
     @overload
     def __new__(
         cls,
         data: DatetimeIndex | Sequence[Timestamp | np.datetime64 | datetime],
         index: Axes | None = ...,
-        dtype=...,
+        dtype: Dtype = ...,
```

Comment:
If `data` represents times, then the values of `Dtype` are restricted to be `TimestampDtypeArg`